.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thank you for taking the time to contribute to Odin-pantry. Please complete this entire template. -->
+
+## Because
+<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
+
+
+## This PR
+<!-- A bullet point list of one or more items describing the specific changes. -->
+- e.g. Add PR templates
+
+
+## Issue
+<!--
```

Comment:
I've seen a team go so far as to include the `#(issue)` on this section, and give instructions to the submitter to replace `(issue)` so they can see the magic of Github make the connection